docs/manage/requesting_data_as_a_collaborator.md

```diff
@@ -95,16 +46,67 @@ You can also remove the data by using `bin/purge_database_json.py`, which will d
 
-Again, data formats are at 
-https://github.com/e-mission/e-mission-server/tree/master/emission/core/wrapper
-
-Let me (@shankari) know if you have any further questions...
+## Alternative Analysis Methods ##
+
+There are a few ways to explore the data beyond the server.  Generally, these methods require a "mongodump" file -- a collection of data, archived in `.tar.gz` format.  Here are the broad steps you need to take in order to work with this data:
+
+1. **Start Docker**:  Ensure you have docker installed on your machine, and a `docker-compose.yml` file saved to your chosen repository.  The following command should start the development environment:
+    ```bash
+    $ docker-compose -f [example-docker-compose].yml up
+    ```
+    Example docker config files can be found in the server repository [here](https://github.com/e-mission/e-mission-server/blob/d2f38bc18d5c415888451e7ad98d40325a74c999/emission/integrationTests/docker-compose.yml#L4). The general construction of a compose file is as follows:
+
+    ```yml
+    version: "3"
+    services:
+    db:
+        image: mongo:4.4.0
+        volumes:
+        - mongo-data:/data/db
+        networks:
+        - emission
+        ports:
+        - "27017:27017" # May change depending on repo
+
+    networks:
+    emission:
+
+    volumes:
+    mongo-data:
+    ```
+2. **Load your data**:  There are a few ways to go about this:
+    - Certain repositories will have a `load_mongodump.sh` script.  Given the correct docker was started in the previous step, this should load all of the data for you.  
+        - Depending on the data being analyzed, loading the entire mongodump may take a _very_ long time.  Ensure that docker's resources are properly increased, and ample time is set aside for the loading process.
+    - If a portion of data is needed, the mongodump unzipped, and its individual components loaded into the docker.
+        - First, unpack your mongo dump file by running `tar -xvf [your_mongo_dump.tar.gz]`
+        - Navigate to the unzipped folder.  Create a new directory, `./dump/Stage_database/`.  Copy your data files into this new directory.
+        - Copy the new `./dump/Stage_database` directory into your Docker's `/tmp/` directory.  This can be done by dragging and dropping the directory via the Docker Desktop client, or done via the command line.
+        - Using the following commands, connect to your docker image, 
+            ```bash
+            $ docker exec -it [your_docker_image_name] /bin/bash
+            root@12345:/ cd tmp; mongorestore
+            ```
+        - More information on this approach can be found in the public dashboard [ReadMe](https://github.com/e-mission/em-public-dashboard/blob/main/README.md#large-dataset-workaround).
+
+
+In general, it is best to follow the instructions of the repository you are working with. There are subtle differences between them, and these instructions are intended as general guidance only.
```

Comment:
We should unify these but obviously we should keep this documentation until we do.